packages/web-react/src/components/FieldGroup/__tests__/FieldGroup.test.tsx

```diff
@@ -24,16 +24,20 @@ describe('FieldGroup', () => {
   validationTextPropsTest(FieldGroup, '.FieldGroup__validationText');
 
   stylePropsTest(
-    (props) => <FieldGroup {...props} label="Label" id="FieldGroupExample" data-testid="test-FieldGroup" />,
+    (props) => <FieldGroup {...props} label="Label" id="field-group-example" data-testid="test-FieldGroup" />,
     'test-FieldGroup',
   );
 
   restPropsTest((props) => <FieldGroup {...props} label="Label" />, 'fieldset');
 
   it('should render items as children', () => {
-    const dom = render(<FieldGroup label="Label">{itemList}</FieldGroup>);
+    render(
+      <FieldGroup id="example-field-group" label="Label">
+        {itemList}
+      </FieldGroup>,
+    );
 
-    const fieldGroup = dom.container.querySelector('fieldset') as HTMLElement;
+    const fieldGroup = screen.getByRole('group') as HTMLElement;
```

Comment:
Is `as HTMLElement` necessary here? In other modifications you are not using it

Reply:
I just forgot to delete it  😄 I think that is not necessary to use HTML types in that case